_opensearch/install/tar.md

```diff
@@ -7,144 +7,455 @@ nav_order: 50
+      hostname opensearch-knn                       2.1.0.0
+      hostname opensearch-ml                        2.1.0.0
+      hostname opensearch-notifications             2.1.0.0
+      hostname opensearch-notifications-core        2.1.0.0
+      hostname opensearch-observability             2.1.0.0
+      hostname opensearch-performance-analyzer      2.1.0.0
+      hostname opensearch-reports-scheduler         2.1.0.0
+      hostname opensearch-security                  2.1.0.0
+      hostname opensearch-sql                       2.1.0.0
+      ```
+
+## Setup OpenSearch in Your Environment
+
+Users that do not have prior experience with OpenSearch may want a list of recommended settings to get started with the service. By default, OpenSearch is not bound to a network interface and cannot be reached by external hosts. Additionally, security settings are either undefined (greenfield install) or are populated by default usernames and passwords if you ran the security demo script by invoking `opensearch-tar-install.sh`. The following recommendations will enable a user to bind OpenSearch to a network interface, create and sign TLS certifications, and configure basic authentication.
+
+The following recommended settings will allow users to:
+
+- Bind OpenSearch to an IP or network inteface on the host.
+- Set initial and max JVM heap sizes.
+- Define an environment variable that points to the bundled JDK.
+- Configure your own TLS certificates - no third-party Certificate Authority required.
+- Create an admin user with a custom password.
+
+If you ran the security demo script then you will need to manually reconfigure settings that were modified. Refer to Security Plugin [Configuration]({{site.url}}{{site.baseurl}}/opensearch/configuration/) for guidance before proceeding.
+{:.note}
+
+Before modifying any configuration files, it's always a good idea to save a backup copy before making changes. The backup file can be used to revert any issues caused by a bad configuration.
+{:.tip}
+
+1. Open `opensearch.yml`.
+   ```bash
+   vi /path/to/opensearch-{{site.opensearch_version}}/config/opensearch.yml
+   ```
+1. Add the following lines.
+   ```bash
+   # Bind OpenSearch to the correct network interface. Use 0.0.0.0
+   # to include all available interfaces or specify an IP addresss
+   # assigned to a specific interface.
+   network.host: 0.0.0.0
+
+   # Unless you have already configured a cluster, you should set
+   # discovery.type to single-node or the bootstrap checks will
+   # fail when you try to start the service.
+   discovery.type: single-node
+
+   # If you previously disabled the security plugin in opensearch.yml,
+   # be sure to re-enable it. Otherwise you can skip this setting.
+   plugins.security.disabled: false
+   ```
+1. Save your changes and close the file.
+1. Specify an initial and max JVM heap size.
+   1.  Open `jvm.options`.
+         ```bash
+         vi /path/to/opensearch-{{site.opensearch_version}}/config/jvm.options
```

Comment:
We should standardize how we refer to paths. I think an example would be nice here as well...

Reply:
I asked about `/path/to` and was told it's acceptable.